dao/README.md

```diff
@@ -0,0 +1,119 @@
+
+Becoming part of the Croncat DAO is much different than token based DAOs. It requires real interaction and participation in community development, governance, research & network growth. To maintain a seat on Croncat DAO, you believe in the vision, aspire to further the mission statement, and generally provide your personal perspective to create positive outcomes for the Croncat community as a whole. Maintaining a seat on the DAO requires interactions like voting at minimum 5 times per year.
+
+# Governance & Operations
+### Council Responsibilities
+DAO council will be the sole responsible entity for the development and promotion of croncat. The council will be made up of different types of persons, each bringing a unique viewpoint to the governance process to align and balance the DAO. You can think of the council representing 3 core entities: Founders, Producers, Consumers. The council will have the power to enact proposals, fund development & marketing, provide guidance on integrations and onboard further community members.
+
+Council members are responsible to the DAO, and have a requirement to on-chain activity periodically to maintain their council seat. By staying active on the chain activities, members not only keep the community inline with their perspective, but also help create a wider base for decisions. Council members are also responsible for doing the research and diligence necessary to make sound decisions for fund allocations.
+
+### Role Definitions
+
+| Role | Capabilities | Definition & Perks |
+|---|---|---|
+|Founder|Proposal, Voting, Treasury, Core|Maintains council members, directs treasury funds towards development initiatives, maintains upgrades. Can vote on all types of proposals. Receives epoch based retainer stipend of a percentage on earned interest balance.|
+|Application|Proposal, Voting|Proposes needs for application integrations, fees or similar. Can vote on operation & cost proposals. Early integration partners receive special swag & NFT.|
+|Agent|Proposal, Voting|Proposes needs for agent runtime, reward amount or similar. Can vote on operation & cost proposals. Early adopters receive special swag & rare NFT.|
+|Commander|Proposal|Proposes reimbursements, work initiatives, development bounties, marketing initiatives & other types of works created for furthering the growth of cron. Commanders receive differing levels of access to things like social, discussion, development session & more based on longevity and work completed.|
+
+### Proposal Types
+* **Treasury Proposal**: Items relating to or including fund or token allocation & staking accounting. 
+* **Operation Cost Proposal**: Items that pay individuals for a certain finalized development or marketing initiative.
+* **Custom Operation**: Special contract function calls that can include core runtime settings, interacting with other dApps or upgrading core contracts.
```

Comment:
This is great. Another proposal type I could see:
**Enhancing incentive economics**: Proposals that identify and provide clear suggestions on how economics can be improved for the Croncat community.

Reply:
Luvvit!